docs/tutorials/projects/guardian.md

```diff
@@ -126,24 +126,26 @@ Go to the **Setup** tab of your new robot's page and follow the steps [to instal
 {{% tab name="Builder UI" %}}
 
 Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
-Click on the **Components** subtab and navigate to the **Create component** menu.
+Click on the **Components** subtab.
 
 1. **Add the board.**
 
-    Enter `local` for the name for your [board component](/components/board/), select the type `board`, and select the `pi` model.
-    Then click **Create component**.
+    Click **Create component** in the lower-left corner of the page.
+    Select `board` for the type, then select `pi` for the model.
+    Enter `local` for the name for your [board component](/components/board/), then click **Create**.
 
 2. **Add the camera.**
 
-    Create a [camera component](/components/camera/) with the name `cam`, the type `camera` and the model `webcam`.
-    Click **Create Component** to add the camera.
+    Click **Create Component** to add the [camera](/components/camera/).
+    Choose type `camera`, model `webcam`, and for the name enter `cam`.
+    Click **Create**.
```

Comment:
```suggestion
    Click **Create component** in the lower-left corner of the page.
    Select `camera` for the type, then select `webcam` for the model.
    Enter `cam` as the name for your [camera component](/components/camera/), then click **Create**.
```

Reply:
Part of that would duplicate the line above so I'm thinking
```suggestion
    Select `camera` for the type, then select `webcam` for the model.
    Enter `cam` as the name for the camera, then click **Create**.
```